src/main/java/edu/greenblitz/robotName/subsystems/Arm/Elbow.java

```diff
@@ -0,0 +1,100 @@
+package edu.greenblitz.robotName.subsystems.Arm;
+
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.greenblitz.robotName.subsystems.Arm.ElbowUtils.ElbowInputsAutoLogged;
+import edu.greenblitz.robotName.subsystems.Arm.ElbowUtils.FalconElbow.FalconElbowConstants;
+import edu.greenblitz.robotName.subsystems.Battery;
+import edu.greenblitz.robotName.utils.GBSubsystem;
+import org.littletonrobotics.junction.Logger;
+import edu.greenblitz.robotName.subsystems.Arm.ElbowUtils.ElbowFactory;
+import edu.greenblitz.robotName.subsystems.Arm.ElbowUtils.IElbow;
+import edu.greenblitz.robotName.subsystems.Arm.ElbowUtils.ElbowConstants;
+public class Elbow extends GBSubsystem {
+
+    private static Elbow instance;
+
+    private IElbow elbow;
```

Comment:
I find it kind of confusing that the elbow subsystem also contains an `IElbow` called `elbow`

Reply:
why, the IElbow present the elbow and all the elbow happen on this elbow

Reply:
> I find it kind of confusing that the elbow subsystem also contains an `IElbow` called `elbow`

its convention